athena-federation-sdk/src/test/java/com/amazonaws/athena/connector/lambda/handlers/CompositeHandlerTest.java

```diff
@@ -107,7 +107,7 @@ public void setUp()
 
         when(mockMetadataHandler.doListTables(any(BlockAllocatorImpl.class), any(ListTablesRequest.class)))
                 .thenReturn(new ListTablesResponse("catalog",
-                        Collections.singletonList(new TableName("schema", "table"))));
+                        Collections.singletonList(new TableName("schema", "table")), null));
```

Comment:
would it be more useful to have this test be paginated?

Reply:
This test only checks that doListTables was called, and null is a valid nextToken.

Reply:
ack.